js/dl.js

```diff
@@ -0,0 +1,20 @@
+const students = [
+  { name: 'Ольга', gender: 'female' },
+  { name: 'Олег', gender: 'male' },
+  { name: 'Кристи', gender: 'female' },
+];
+
+function sortByGender(students) {
+  let maleArray = [];
+  let femaleArray = [];
+  students.forEach(student => {
+    if (student.gender === 'male') {
+      maleArray.push(student);
+    } else {
+      femaleArray.push(student);
+    }
+  });
+  return [...maleArray, femaleArray];
```

Comment:
Результатом должен быть объект типа: 
```suggestion
const result = {
    male: [{name: 'Oleh', gender: 'male'}],
    female: [{name: 'Olha', gender: 'female'}, {name: 'Kristi', gender: 'female'}]
}
```
У тебя получается массив. 
Что бы пофиксить это, будет достаточно этого кода:
```suggestion
  return {
      female: femaleArray,
      male: maleArray
      }
```